debian/control

```diff
@@ -24,7 +24,6 @@ Build-Depends:
  libcal-dev (>= 0.2.4),
  libdevlock-dev,
  libupower-glib-dev (>=1:0.99.7.5),
- libsystemd-dev,
```

Comment:
Who added that in the first place?!? ACK, but why is this commit part of that PR?

Reply:
afaik pali added this. He added some systemd code to mce, not sure what it dose. It is however behind a ifdef and the packaging scripts do not set this define. therefore this dependancy is invalid.

I thought it a to insignificant change to push as its own pr, but i dont care, it can be one.

Reply:
ok, lets leave it as it is

Reply:
I think Spinal added it.